draft-ietf-scitt-architecture.md

```diff
@@ -179,19 +179,19 @@ Consumer of Signed Statements:
 Envelope:
 
 : metadata and an Issuer's signature is added to a Statement via a COSE Envelope by the Issuer to produce a Signed Statement.
-An Envelope contains the identity of the Issuer and other information to help components responsible for validation that are part of a Transparency Services to identify the software Artifact referred to in a Signed Statement.
+The Envelope contains the identity of the Issuer and information about the Artifact to help Transparency Services to properly validate and process the Signed Statement.
```

Comment:
I am leery of changing this to information about the Artifact.  The job of the transparency service is to countersign the signature not make any determination or claims about the content.  In fact there are scenarios where we don't want that information to be anywhere near the service.

Reply:
I believe this update is saying the registration policy can read the non-opaque envelope header to make the decisions.